src/utils/totalTryInput.js

```diff
@@ -0,0 +1,14 @@
+import { Console } from "@woowacourse/mission-utils";
+const totalTryInput = async () => {
+  const total = await Console.readLineAsync("시도할 횟수는 몇 회인가요?\n");
+
+  if (!total || total.trim() === "") {
+    throw new Error(ERROR_MESSAGES.EMPTY_INPUT);
+  } else if (isNaN(total)) {
+    throw new Error(ERROR_MESSAGES.NOT_A_NUMBER);
+  } else if (total < 1) {
+    throw new Error(ERROR_MESSAGES.POSITIVE_NUMBER);
+  }
+  return total;
```

Comment:
totalTryInput에 대한 유효값 검사를 따로 빼도 좋을 것 같아요. 
또한 if 를 사용해서 가독성을 올리는 거 어떤가요 ?!
```suggestion
  if (!total || total.trim() === "") {
    throw new Error(ERROR_MESSAGES.EMPTY_INPUT);
  } 
  if (isNaN(total)) {
    throw new Error(ERROR_MESSAGES.NOT_A_NUMBER);
  } 
  if (total < 1) {
    throw new Error(ERROR_MESSAGES.POSITIVE_NUMBER);
  }
  return total;
```